lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -43,25 +44,20 @@
 
 <div class="create-team form-actions">
     <button class="action action-primary">
-        <%=
-            interpolate_text(
-                _.escape(gettext("{primaryButtonTitle} {span_start}a team{span_end}")),
-                {
-                    'primaryButtonTitle': primaryButtonTitle, 'span_start': '<span class="sr">', 'span_end': '</span>'
-                }
-            )
-        %>
+        <span aria-hidden="true"><%- primaryButtonTitle %></span>
+        <% if (action === 'create') { %>
+            <span class="sr"><%- gettext("create team.") %></span>
```

Comment:
I'm not sure if we have a convention, it just seems weird. One thing is
that 'create team' and 'Create Team' would be different strings to
translate. I advocate using the same casing everywhere, for consistency and
to avoid extra work that we make for the translators.

On Mon, Aug 31, 2015 at 10:51 AM, Muhammad Ammar <notifications@github.com>
wrote:

> In lms/djangoapps/teams/static/teams/templates/edit-team.underscore
> <https://github.com/edx/edx-platform/pull/9513#discussion_r38319282>:
>
> > @@ -43,25 +44,20 @@
> >
> >  <div class="create-team form-actions">
> >      <button class="action action-primary">
> > -        <%=
> > -            interpolate_text(
> > -                _.escape(gettext("{primaryButtonTitle} {span_start}a team{span_end}")),
> > -                {
> > -                    'primaryButtonTitle': primaryButtonTitle, 'span_start': '<span class="sr">', 'span_end': '</span>'
> > -                }
> > -            )
> > -        %>
> > +        <span aria-hidden="true"><%- primaryButtonTitle %></span>
> > +        <% if (action === 'create') { %>
> > +            <span class="sr"><%- gettext("create team.") %></span>
>
> @sarina <https://github.com/sarina> Does the case of of text matters for
> screen reader?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/9513/files#r38319282>.
>